packages/get-starknet/public/index.html

```diff
@@ -0,0 +1 @@
+<html></html>
```

Comment:
What is the purpose of this HTML file ?
Can you add come comments in the Code file to describe its need and usage ?

Reply:
good catch , it should be deleted, i will remove them

Reply:
it is to test the module fed working fine or not, i will keep it